app/brave_generated_resources.grd

```diff
@@ -289,6 +289,9 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_NEW_TAB_GEMINI" desc="The label for Gemini show/hide feature toggle">
         Gemini
       </message>
+      <message name="IDS_SETTINGS_NEW_TAB_BITCOIN_DOT_COM" desc="The label for Bitcoin.com show/hide feature toggle">
+        Bitcoin.com
+      </message>
```

Comment:
I think it's better to move this to the component strings mentioned below

Reply:
moved

Reply:
I did so for the other strings, but I see this one doesn't include any `<part...>` files, would I need to make a separate one just for this entry? Or is this one fine in here for now

Reply:
why can't it go in the component part file? We don't want string's when the build flag is not defined.

Reply:
moved